airgun/views/settings.py

```diff
@@ -9,6 +9,8 @@
 
 class SettingsView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Settings']")
+    email_button = Text('#test_mail_button')
+    notification_locator = Text('//*[@id="toast-notifications-container"]/div/div')
```

Comment:
try not to use the * instead that use the HTML tag

Reply:
Fixed